### ISSUE TITLE
A11y: Mobile filter panel - Visual headline isn't semantically marked up

### ISSUE THREAD

**Issue by timo-haas-gh** [shopware/platform#5721]
*PHP Version:* 8.3   | *Shopware Version:* 6.6.8.2  | *Affected area / extension:* Platform(Default) 

 ----

*_Actual behaviour:_*

The mobile filter panel is visually created as a heading, but semantically the text is in a div element.
https://github.com/shopware/shopware/blob/trunk/src/Storefront/Resources/views/storefront/component/listing/filter-panel.html.twig#L4

![Image](https://github.com/user-attachments/assets/cf29ec21-f363-4d87-9b97-b8c9bfa4925b)

*_Expected behaviour:_*

Should be marked up correctly for accessibility.

*_How to reproduce:_*

Open up a listing with filter options in mobile view.

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39855**



[created from NEXT-39855, comment 518491]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.0**

 This issue will be / was solved in Shopware version ***6.6.10.0***.



[created from NEXT-39855, comment 523139]
